src/NServiceBus.Testing/ExpectedInvocations/ExpectHandleCurrentMessageLater.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NServiceBus.Testing.ExpectedInvocations
+{
+    class ExpectHandleCurrentMessageLater : ExpectInvocation
+    {
+        public override void Validate(TestableMessageHandlerContext context)
+        {
+            if (!context.HandleCurrentMessageLaterWasCalled)
+            {
+                Fail(context.SentMessages);
```

Comment:
Feels like this should be an entirely different text here since it's not related to outgoing messages at all?